src/lib/converter/converter.ts

```diff
@@ -134,11 +140,13 @@ export class Converter extends ChildableComponent<
      * Compile the given source files and create a project reflection for them.
      *
      * @param entryPoints the entry points of this program.
-     * @param program the program to document that has already been type checked.
+     * @param programs the programs to document, that have already been type checked.
+     * @param packages an array of packages (used in --packages mode)
      */
     convert(
         entryPoints: readonly string[],
```

Comment:
I'm not that happy with the way in which I've plumbed the packages as a third param to convert where you're basically either using the first 2 params or the third. However I wasn't really sure if it would make sense to support passing non-package entry-points as well as packages simultaneously in to this system, or if these data structures could be unified earlier in the processing to make this less weird.

Looking for ideas here on better architecture.

Reply:
I think the way I'd like to handle this is to make a breaking change, and accept an array of `DocumentationEntrypoint`, then the caller is responsible for turning an array of programs (in the entryPoint case) or packages into the appropriate structure.

Means this can't be released without a minor version bump, but I think this feature deserves one anyways - I'll take what I've done towards 0.21, merge it with this, and call the theme refactor 0.22 rather than making this wait.